helios-services/src/main/java/com/spotify/helios/master/resources/HostsResource.java

```diff
@@ -122,11 +123,14 @@ public HostsResource(final MasterModel model) {
           })
           .collect(Collectors.toList());
 
+      final Function<String, Map<String, String>> getLabels =
```

Comment:
this might be easier to read as a private method rather than an inline function, since this doesn't close over anything in the current method body